cloudflare/resource_cloudflare_waiting_room_event.go

```diff
@@ -0,0 +1,205 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareWaitingRoomEvent() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareWaitingRoomEventCreate,
+		Read:   resourceCloudflareWaitingRoomEventRead,
+		Update: resourceCloudflareWaitingRoomEventUpdate,
+		Delete: resourceCloudflareWaitingRoomEventDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareWaitingRoomEventImport,
+		},
+
+		Schema: resourceCloudflareWaitingRoomEventSchema(),
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Second),
+			Update: schema.DefaultTimeout(30 * time.Second),
+		},
+	}
+}
+
+func expandWaitingRoomEvent(d *schema.ResourceData) (cloudflare.WaitingRoomEvent, error) {
+	var disableSessionRenewal *bool
+	switch b := d.Get("disable_session_renewal").(type) {
+	case bool:
+		disableSessionRenewal = &b
+	case nil:
+		disableSessionRenewal = nil
+	}
```

Comment:
i think we can swap this for a `d.GetOkExists` and the value can use `cloudflare.BoolPtr` to take the boolean and return a *bool.

Reply:
It looks like `d.GetOkExists` is deprecated and might be removed. Would just using  `d.GetOk` be sufficient?